source/components/inputs/input.ts

```diff
@@ -36,6 +39,10 @@ export class InputComponent<T> implements AfterViewInit, OnInit {
 		if (this.object.isNullOrEmpty(this.name)) {
 			this.name = this.inputType + '-' + this.guid.random();
 		}
+
+		if (!this.object.isNullOrEmpty(this.value)) {
```

Comment:
We decided to move away from this. Instead you can just use truthy here, as in `if (this.value)`